assignments/arrays.js

```diff
@@ -63,34 +63,77 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+for(let i = 0; i < inventory.length; i++){
+    if(inventory[i].id === 33){
+console.log(`\nChallenge 1 Output: \nCar 33 is a ${inventory[i].car_year} ${inventory[i].car_make} ${inventory[i].car_model}` );
+    }
+}
 
 
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+let lastCar = inventory.pop();
+
+console.log(`\nChallenge 2 Output: \n${lastCar.car_make} ${lastCar.car_model}`);
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
 let carModels = [];
-console.log();
+//'riolet was lowercase and missed the sorting
+//Turning all first letters uppercase, then sorting and logging to console.
+for(let i = 0; i < inventory.length; i++){
+    carModels[i] = inventory[i].car_model; 
+    let model = carModels[i].split('');  
+    model.splice(0,1,model[0].toUpperCase());   
+    carModels.splice(i,1,model.join(''));
+}
+carModels.join('');
+carModels.sort();
```

Comment:
Sort is a powerful method and it is important to know how any function works underneath the hood.